### PR TITLE
arome/28690/Updated partners page

### PR DESCRIPTION
**Pull request by arome-binary**
Changes:

-   Added links to developers page on footer and mobile menus
-   Added card and mobile menu link on partners page directing to developers page
-   Minify menu SVGs

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — APPROVED** [review]

**Review by yashim-deriv — APPROVED** [review]
LGTM. Is the Navigation Label `API` intended?

File: src/pages/partners/_partnership-opportunities.js [hunk 1]
```diff
@@ -92,20 +102,26 @@ const PartnershipOpportunities = () => (
...
-                        {localize('Developer programme')}
+                        {localize('API')}
                     </Header>
-                    <WithdrawIcon />
+                    <DeveloperProgramme />
                 </Flex>
                 <Text>
                     {localize(
-                        'Develop your own app or interface on top of our trading infrastructure. Earn affiliate commission from new signups, or mark up the price of the contracts traded in your app.',
+                        'Launch your trading app powered by our APIs. Get up to 5% in markup earnings from every trade your clients make on your app.',
```
Comment (kevinw-binary) [inline src/pages/partners/_partnership-opportunities.js:121]: Do we need to make PR to crowdin instead since it has new string ?

**Review by kevinw-binary — COMMENTED** [review]
Added a comment regarding new string

File: src/pages/partners/_partnership-opportunities.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/partners/_partnership-opportunities.js:121]: Nope we are releasing it straight to master first.

**Review by kevinw-binary — APPROVED** [review]

**Review by suthesan-deriv — APPROVED** [review]

**Review by suthesan-deriv — APPROVED** [review]

**Pull request merged**